source/rust.txt

```diff
@@ -37,35 +37,71 @@ See `Installation <https://github.com/mongodb/mongo-rust-driver#Installation>`__
 Connect to MongoDB Atlas
 ------------------------
 
+Select from the :guilabel:`Sync` or :guilabel:`Async` tabs below for
+corresponding connection code samples.
+
 .. include:: /includes/atlas-connect-blurb.rst
 
-.. code-block:: rust
+.. tabs::
+
+   .. tab:: Sync
+      :tabid: rust-sync
+
+      .. code-block:: rust
+
+         // Be sure to set default-features=false and enable the "sync" feature in your Cargo.toml
+         use mongodb::{bson::doc, sync::Client};
+
+         fn main() -> mongodb::error::Result<()> {
+             // Get a handle to the deployment.
```

Comment:
nit: is "deployment" a term we're using commonly across the driver docs? I would expect this to be "cluster" instead.

Reply:
I know choosing terms is on Tony Sansone's task list, but I agree that cluster makes more sense especially since it's the term used in the connection string. Will update all instances.